kpops/components/streams_bootstrap/producer/model.py

```diff
@@ -4,14 +4,15 @@
     KafkaAppValues,
     KafkaStreamsConfig,
 )
+from kpops.components.streams_bootstrap import StreamsBootstrapValues
 from kpops.utils.docstring import describe_attr
 
 
 class ProducerStreamsConfig(KafkaStreamsConfig):
     """Kafka Streams settings specific to Producer."""
 
 
-class ProducerAppValues(KafkaAppValues):
+class ProducerAppValues(StreamsBootstrapValues, KafkaAppValues):
```

Comment:
why does the generic producer have to inherit from streams bootstrap? 🤔

Reply:
I couldn't find another way to pass the `imageTag` to the `ProducerAppValues`.
The class `ProducerApp` inherits `KafkaApp` and `StreamsBootstrap`. I think the same logic makes sense for the `ProducerAppValues`.

Reply:
Ah you're right, the producer is actually not generic but specific to streams bootstrap